test/regression.py

```diff
@@ -668,6 +668,14 @@ def testGAMESS_WinGAMESS_dvb_td_trplet_2007_03_24_r1_out(logfile):
         parse_version(logfile.data.metadata["package_version"]), Version
     )
 
+def testnoparseGAMESS_WinGAMESS_H2O_def2SVPD_triplet_2019_06_30_R1_out(filename):
+    """Check if the molden writer can handle an unrestricted (ROHF) case
+    """
+    data = cclib.io.ccread(os.path.join(__filedir__,filename))
+    writer = cclib.io.moldenwriter.MOLDEN(data)
+    # Check size of Atoms section.
+    self.assertEqual(len(writer._mo_from_ccdata()), ((4+len(data.gbasis))*len(data.mocoeffs)))

```

Comment:
```diff
diff --git a/test/regression.py b/test/regression.py
index 23585ae9..78725434 100644
--- a/test/regression.py
+++ b/test/regression.py
@@ -69,7 +69,7 @@ from cclib.parser import Psi4
 from cclib.parser import QChem
 from cclib.parser import Turbomole
 
-from cclib.io import ccopen
+from cclib.io import ccopen, ccread, moldenwriter^M
 
 # This assume that the cclib-data repository is located at a specific location
 # within the cclib repository. It would be better to figure out a more natural
@@ -671,10 +671,10 @@ def testGAMESS_WinGAMESS_dvb_td_trplet_2007_03_24_r1_out(logfile):
 def testnoparseGAMESS_WinGAMESS_H2O_def2SVPD_triplet_2019_06_30_R1_out(filename):
     """Check if the molden writer can handle an unrestricted (ROHF) case
     """
-    data = cclib.io.ccread(os.path.join(__filedir__,filename))
-    writer = cclib.io.moldenwriter.MOLDEN(data)
-    # Check size of Atoms section.
-    self.assertEqual(len(writer._mo_from_ccdata()), ((4+len(data.gbasis))*len(data.mocoeffs)))
+    data = ccread(os.path.join(__filedir__,filename))^M
+    writer = moldenwriter.MOLDEN(data)^M
+    # Need to account for the Sym=, Ene=, Spin=, and Occup= lines.^M
+    assert len(writer._mo_from_ccdata()) == (4 + data.nbasis) * data.nmo^M
```

Reply:
Maybe it's also worth checking the value of a single beta element in the `_mo_from_ccdata()` output.